ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.19__amd64-nightly-4.19-upgrade-from-stable-4.18.yaml

```diff
@@ -471,15 +471,6 @@ tests:
     test:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-ipi-marketplace
-- as: azure-ipi-proxy-workers-rhcos-rhel8-f28
```

Comment:
Suggest to replace it with `azure-ipi-proxy`

Reply:
@jianlinliu there's already `azure-ipi-fullyprivate-proxy` for this upgrade path
```
openshift-openshift-tests-private-release-4.19__multi-stable-4.19-upgrade-from-stable-4.18.yaml:- as: azure-ipi-fullyprivate-proxy-arm-f28
```

Reply:
`proxy` v.s `fullyprivate-proxy`, personally I prefer to test it for more coverage, also to make upgrade ci align with e2e ci.